text/0000-cargo-prepublish.md

```diff
@@ -112,6 +120,30 @@ With this setup, the dependency graph for Servo will contain *two* versions of
 `0.9.1` is considered a minor release against `0.9.0`, while `0.9.0` and `0.8.0`
 are incompatible.
 
+### Scenario: augmenting with a bugfix
+
+Let's say that while developing `foo` we've got a lock file pointing to `xml-rs`
+`0.8.0`, and there's an `0.8.0` branch of `xml-rs` that hasn't been touched
```

Comment:
Wasn't `foo` supposed to depend on version `0.9.0`? It's `baz` that depends on `0.8.0`. Is this just a typo?